src/core_plugins/kibana/public/management/sections/indices/create_index_pattern_wizard/components/step_index_pattern/components/loading_indices/loading_indices.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+import {
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiText,
+  EuiTextColor,
+} from '@elastic/eui';
+
+export const LoadingIndices = () => (
+  <EuiFlexGroup justifyContent="center">
+    <EuiFlexItem grow={false}>
+      <EuiText>
+        <EuiTextColor color="subdued">
+          Looking for matching indices...
```

Comment:
Can we treat this loading feedback the same way we treated the feedback the same way we treat the loading feedback when [checking for indices](https://github.com/elastic/kibana/blob/master/src/core_plugins/kibana/public/management/sections/indices/create_index_pattern_wizard/create_index_pattern_wizard.html#L65)? With a spinner, left-aligned, etc?

Reply:
Yup!